.clasprc.json.enc

```diff
@@ -0,0 +1,2 @@
+�t�@�NM����.|9%�	�M�mc)eJ�&���,nv�9��N����7=b����C,P8�O��A��T�,SЇ	��f�4wk�gn�<��2�p9p��ץ�p�Y�d|K�Z(�xn�T(��"��g���={)3�(�y��O���S��

```

Comment:
Can we move this file to `tests/` and update the relative path in the `.travis.yaml`?
- This file is only used in our tests
- Let's try to reduce the number of top-level files/folders.